jobs/tasks.py

```diff
@@ -2,18 +2,21 @@
 
 from core.core import calculate
 from jobs.models import Job, CREATED, RUNNING, COMPLETED, ERROR
+from predModels.models import ModelSplit, PredModels
+from sklearn.externals import joblib
 
 
 @job("default", timeout='1h')
 def prediction_task(job_id):
     print("Start prediction task ID {}".format(job_id))
     job = Job.objects.get(id=job_id)
-
     try:
         if job.status == CREATED:
             job.status = RUNNING
             job.save()
-            result = calculate(job.to_dict())
+            result,split = calculate(job.to_dict())
```

Comment:
Does this mean that the split is always created? I'm assuming this contains the models. So it just doesn't save it always.

What is the performance impact of always generating a split?